lib/Minion/Backend.pm

```diff
@@ -589,6 +589,12 @@ Number of times performing this job will be attempted.
 
 Delay job for this many seconds (from now), defaults to C<0>.
 
+=item parents
+
+  parents => [$id1, $id2, $id3]
+
+Jobs this job depends on.
```

Comment:
This is not the only place the new option needs to be documented.

Reply:
Added it lib/Minion/Backend/Pg.pm

Reply:
Also `Minion::Job`.

Reply:
Added, sorry >.<